llvm/include/llvm/CodeGen/VirtRegMap.h

```diff
@@ -49,7 +48,7 @@ class TargetInstrInfo;
     /// it; even spilled virtual registers (the register mapped to a
     /// spilled register is the temporary used to load it from the
     /// stack).
-    IndexedMap<Register, VirtReg2IndexFunctor> Virt2PhysMap;
+    IndexedMap<MCRegister, VirtReg2IndexFunctor> Virt2PhysMap;
```

Comment:
If the key is a virtual register, this should still be Register

Reply:
The Key is determined by the `argument_type` field in `VirtReg2IndexFunctor`.